exporter/coralogixexporter/metrics_client.go

```diff
@@ -85,11 +86,19 @@ func (e *metricsExporter) pushMetrics(ctx context.Context, md pmetric.Metrics) e
 		resourceMetric.Resource().Attributes().PutStr(cxSubsystemNameAttrName, subsystem)
 	}
 
-	_, err := e.metricExporter.Export(e.enhanceContext(ctx), pmetricotlp.NewExportRequestFromMetrics(md), e.callOptions...)
+	resp, err := e.metricExporter.Export(e.enhanceContext(ctx), pmetricotlp.NewExportRequestFromMetrics(md), e.callOptions...)
 	if err != nil {
 		return processError(err)
 	}
 
+	partialSuccess := resp.PartialSuccess()
+	if !(partialSuccess.ErrorMessage() == "" && partialSuccess.RejectedDataPoints() == 0) {
```

Comment:
Could we only check for `partialSuccess.RejectedDataPoints() != 0`? 🤔  

I'm assuming all partial successes will always have some rejected data points and I'm not sure if this is true.

Reply:
I followed what is implemented for OTLP exporter. I think there can be situations where no telemetry data will be rejected but the backend can send a partial success error message (depending on the backend).